src/pages/node-express/lesson11-a1.mdx

```diff
@@ -0,0 +1,948 @@
+  email1.value = null;
+  password1.value = null;
+  password2.value = null;
+  setDiv(registerDiv);
+};
+```
+
+The `login.js` module is as follows:
+
+```js
+import {
+  inputEnabled,
+  setDiv,
+  token,
+  message,
+  enableInput,
+  setToken,
+} from "./index.js";
+import { showLoginRegister } from "./loginRegister.js";
+import { showJobs } from "./jobs.js";
+
+let loginDiv = null;
+let email = null;
+let password = null;
+
+export const handleLogin = () => {
+  loginDiv = document.getElementById("logon-div");
+  email = document.getElementById("email");
+  password = document.getElementById("password");
+  const logonButton = document.getElementById("logon-button");
+  const logonCancel = document.getElementById("logon-cancel");
+
+  loginDiv.addEventListener("click", (e) => {
+    if (inputEnabled && e.target.nodeName === "BUTTON") {
+      if (e.target === logonButton) {
+        showJobs();
+      } else if (e.target === logonCancel) {
+        showLoginRegister();
+      }
+    }
+  });
+};
+
+export const showLogin = () => {
+  email.value = null;
+  password.value = null;
+  setDiv(loginDiv);
+};
+```
+
+The `jobs.js` module is as follows:
+
+```js
+import {
+  inputEnabled,
+  setDiv,
+  message,
+  setToken,
+  token,
+  enableInput,
+} from "./index.js";
+import { showLoginRegister } from "./loginRegister.js";
+import { showAddEdit } from "./addEdit.js";
+
+let jobsDiv = null;
+let jobsTable = null;
+let jobsTableHeader = null;
+
+export const handleJobs = () => {
+  jobsDiv = document.getElementById("jobs");
+  const logoff = document.getElementById("logoff");
+  const addJob = document.getElementById("add-job");
+  jobsTable = document.getElementById("jobs-table");
+  jobsTableHeader = document.getElementById("jobs-table-header");
+
+  jobsDiv.addEventListener("click", (e) => {
+    if (inputEnabled && e.target.nodeName === "BUTTON") {
+      if (e.target === addJob) {
+        showAddEdit(null);
+      } else if (e.target === logoff) {
+        showLoginRegister();
+      }
+    }
+  });
+};
+
+export const showJobs = async () => {
+  setDiv(jobsDiv);
+};
+```
+
+The `addEdit.js` module is as follows:
+
+```js
+import { enableInput, inputEnabled, message, setDiv, token } from "./index.js";
+import { showJobs } from "./jobs.js";
+
+let addEditDiv = null;
+let company = null;
+let position = null;
+let status = null;
+let addingJob = null;
+
+export const handleAddEdit = () => {
+  addEditDiv = document.getElementById("edit-job");
+  company = document.getElementById("company");
+  position = document.getElementById("position");
+  status = document.getElementById("status");
+  addingJob = document.getElementById("adding-job");
+  const editCancel = document.getElementById("edit-cancel");
+
+  addEditDiv.addEventListener("click", (e) => {
+    if (inputEnabled && e.target.nodeName === "BUTTON") {
+      if (e.target === addingJob) {
+        showJobs();
+      } else if (e.target === editCancel) {
+        showJobs();
+      }
```

Comment:
I'm not sure if it was intentional, but the old page had no `else if` code here to handle clicking on the "Cancel" button on the add/edit job page